app/controllers/file_attachments_controller.rb

```diff
@@ -75,7 +75,7 @@ def destroy
     result = FileAttachments::DestroyInteractor.call(params: params, user: current_user)
     attachment_revision = result.attachment_revision
 
-    redirect_to file_attachments_path(params[:document]),
+    redirect_to params[:wizard] == "featured" ? featured_attachments_path(params[:document]) : file_attachments_path(params[:document]),
                 notice: t("file_attachments.index.flashes.deleted",
```

Comment:
We should keep translations specific to the screen on which they're shown. So in this case, the translation should be `attachments.index.flashes.deleted`, or similar. Could we use a separate one here, so they can vary independently? In this case, we still haven't confirmed which flashes for attachments, so we can actually leave it out entirely until that design work is completed.

Reply:
I'll split up the logic, so that the flash message is only shown for inline attachments.